components/x11/libXcursor/Makefile

```diff
@@ -12,41 +12,33 @@
 #
 # Copyright 2015 Aurelien Larcher
 #
+BUILD_BITS=64_and_32
 
 include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
 
-COMPONENT_NAME=           libXcursor
-COMPONENT_VERSION=        1.1.15
-COMPONENT_FMRI=           x11/library/libxcursor
+COMPONENT_NAME=     libXcursor
+COMPONENT_VERSION=  1.2.0
```

Comment:
Why is it major version change? Are there any compatibility issues?

Reply:
I did not find anything suspicious: https://cgit.freedesktop.org/xorg/lib/libXcursor/log/.